CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,14 @@
+# Observatorium Community Code of Conduct
+
+Observatorium follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Additionally, our team members can be contacted directly for code of conduct abuse, as described below.
+
+# Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team members (e.g @bwplotka, @brancz) at [CNCF slack workspace](https://slack.cncf.io/).
```

Comment:
Not sure we need to print any individual user's slack handles here. It may also give the false impression of some particular rule